### PR TITLE
Update Full-stack tutorial links to point to Odyssey

### PR DESCRIPTION
**Pull request by rkoron007**

### CODE REVIEW

File: docs/source/get-started.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Get started with Apollo Client
 
 Hello! This short tutorial gets you up and running with Apollo Client.
 
-> For a more complete introduction to the entire Apollo platform, [check out **Odyssey**, Apollo's new learning platform](https://odyssey.apollographql.com?utm_source=apollo_docs&utm_medium=referral). You can also complete the [full-stack tutorial](/tutorial/introduction/).
+> For a more complete introduction to the entire Apollo platform, [check out **Odyssey**, Apollo's new learning platform](https://odyssey.apollographql.com?utm_source=apollo_docs&utm_medium=referral). You can also complete the [full-stack quickstart](https://www.apollographql.com/tutorials/fullstack-quickstart/introduction).
```
Comment (StephenBarlow) [inline docs/source/get-started.mdx:7]: Now that the full-stack quickstart is on Odyssey, I reckon we can get away with cutting this second sentence entireyl

**Review by StephenBarlow — APPROVED** [review]
Approved with one suggestion!

**Review by benjamn — COMMENTED** [review]
These look like the right URLs, but I'm currently getting 500 errors when I try to visit them (for example, when clicking directly on [this link](https://www.apollographql.com/tutorials/fullstack-quickstart/fetching-data-with-queries)). Are we still waiting for something to be hooked up?

Screen Shot 2022-04-26 at 12 44 20 PM

**Comment by StephenBarlow** [comment]
@benjamn That was a Netlify-migration-caused hiccup that is now under investigation and that link should work as expected now!

**Comment by rkoron007** [comment]
@benjamn Try navigating to that link again? I think Netlify was having some issues a moment ago, but those links _should_ be working now.

**Pull request merged**